ESS/v3/_init.yaml

```diff
@@ -10,4 +10,5 @@ instances:
               device_type: FTDI
               ftdi_id: ABC
               sensor_type: Temperature
+              baud_rate: 19600
```

Comment:
I gently suggest omitting this, since it is the default.

Reply:
Done